src/ape_accounts/config.py

```diff
@@ -0,0 +1,14 @@
+from ape.api.config import PluginConfig
+
+
+class AccountsConfig(PluginConfig):
+    """
+    Config accounts generally.
+    """
+
+    show_full_calldata: bool = False
```

Comment:
I am wondering if we should make this like an enum instead so we can support other forms of calldata, such "decoded"

```
accounts:
  calldata_repr: "full"  #  defaults to "abridged"
```